site/en/guides/porting/define-cmake-rules.md

```diff
@@ -0,0 +1,133 @@
+# Define CMake rules
```

Comment:
Capitalize "rules"

We use title case for h1 headers (page titles), sentence case for everything else